### ISSUE TITLE
Bug report: v.1.16.5-6.1.0

### ISSUE THREAD

**Issue by NesVicOC** [GrowthcraftCE/Growthcraft-1.16#178]
This post is going to be a summary of every bug I have encountered on these first oficial release of the 1.16 port, I will be searching for more bugs after this post, so i´ll be reporting them in the future by replying to this thread.

- #179 A Churn bug is causing this block to neither give you butter nor butter milk after putting cream into it and completing the separating process, it only works the first time the player interacts with it after placing it, sometimes it doesnt even work this way and just makes your cream disappear. 

- #179 After using the Churn and experimenting issues with his funcionality, there´s a chance to not being able to open the GUI from any GC block.

- [Expected] #176 Butter and Butter milk doesnt show any functionality in JEI.

- [Need More Info] After setting the GUI config to true manually, some recipes for the roaster that weren´t there before appear in the JEI.

- #180 When planting Thistle and breaking it (w/o waiting for it to grow), it gives you a fully grown thistle instead of a seed, so players are able to duplicate thistle and avoid waiting for it to grow since the stonecutter gives you two seeds with only one fully grown thistle.
 
- #181 The Cheese vat GUI can´t be opened normally (w/o changing the default config file settings), what causes players to not been able to recover items they throw in it by mistake w/o changing the file.

- #181 The Cheese vat doesnt transform liquids after using the rigth recipe (needs more test for making sure how and when this happens).
 
- [Expected] #176 Culture jar cant make starter culture without putting starter culture inside (so you need starter culture to make starter culture), and the crafting table recipe that uses skim milk and a stick for this item doesn´t give your bucket back (like Vanilla cake recipe does).

- #182 A Pancheon bug is causing this block to show his second level (having 2000 mB inside) when only having one bucket in it. This is caused after putting and taking a milk bucket from it to fast (this is only a visual bug, when taking the milk out it will only give you one bucket).

- #182 Pancheon different levels looks the same, so you dont know when milk turns into Cream and Skim milk.

- #183 When breaking Fruit press from the top block, it wont give you the item back.

- #184 Fermentation barrel can take every item in the game and will start the fermentation process with every item he can take, after finishing the process, if the item isn´t able to fermentate the barrel fluid inside, it will start the process again.

- #185 Fermentation barrel wont be able to fermentate anything if not full, but will start to infinitely try to process anyways.

- [Expected] #176 If the Fermentation barrel is full and you put the rigth Yeast in it, it would take all the yeast you have placed in the barrel (if you have 1 yeast in there, it will take that 1 yeast to complete the fermentation, if you have 64, it will take the 64).

**Comment by NesVicOC** [comment]
This reply extends the actual list of bugs I have found. This time I tested the mod in creative mode and used /randomtickspeed modifications.

- #186 Fruit Press makes a sound when opening the GUI that has an inadequate volume (earrape).

- [Need More Info] When making Cheese, every different recipe leads to making Asiago, so you can make any other type of cheese.

- [Expected] #176 When Cheese is draining with a Pancheon below, particles appear, but the Pancheon never creates any liquid after the draining process concludes.

- #187 Rope doesn´t drop anything when broken.

- #188 Aging Cheese its not working as intended, it sometimes lets you take cheese slices instantly after placement, and sometimes it doesnt let you take any slices until you get out and re-enter your world (no matter how long you wait for it to age).

- [Expected] #176 You aren´t able to Wax the cheese.

- #185 Fermentation barrel sometimes refuse to fermentate fluids (tested a few different things in the creative world and anything seemed to work, while my survival world follows the rules I wrote in the first post).

- [Expected] #176 Apiary doesn´t generate a bee when completely empty, but seems to work perfectly when you take the first bee from creative and put it inside (generates more bees, wax and honey).

**Comment by Alatyami** [comment]
Most of what you are seeing in Creative is expected. When in creative you should not get drops back.

Going to create sub-tickets for the rest and address them individually.

Are you using a modpack?

"When making Cheese, every different recipe leads to making Asiago, so you can make any other type of cheese." 

The mixing vat uses a recipe file. The only way this can happen is if your ingredients match the recipe file. I am not able to duplicate this behavior.

**Comment by Alatyami** [comment]
"After setting the GUI config to true manually, some recipes for the roaster that weren´t there before appear in the JEI." 

Which recipes? The GUI and JEI/recipes have nothing to do with each other, so need to know which items started showing up in JEI.

**Comment by Alatyami** [comment]
`Butter and Butter milk doesnt show any functionality in JEI.`

Interesting fact ... it never has had a purpose. I have plans to move into more cooking. at that point, it will have a purpose.

`Culture jar cant make starter culture without putting starter culture inside (so you need starter culture to make starter culture), and the crafting table recipe that uses skim milk and a stick for this item doesn´t give your bucket back (like Vanilla cake recipe does).`

In order to start growing starter culture and various yeasts, you have to have one already. Not getting a bucket back from the recipe is expected and not a bug. The bucket is a one-time cost as long as you grow your starter cultures. It is intended to promote using the culture jar instead of the crafting table.

`If the Fermentation barrel is full and you put the right Yeast in it, it would take all the yeast you have placed in the barrel (if you have 1 yeast in there, it will take that 1 yeast to complete the fermentation, if you have 64, it will take the 64).`

This is expected as putting too much yeast in a fermenting barrel will spoil the extra yeast and make it not usable.

**Comment by NesVicOC** [comment]
> Most of what you are seeing in Creative is expected. When in creative you should not get drops back.

I have wrongly expressed myself, I was talking about taking all the machines and items from there just for focusing into testing, everything was made on survival, the only creative thing I did was building the vats and machines before using them on survival.

> Are you using a modpack?

Yes, a custom one. If you think this could be changing some of the issues, I will do a testing w/o the other mods.

> The mixing vat uses a recipe file. The only way this can happen is if your ingredients match the recipe file. I am not able to duplicate this behavior.

I know, that´s the issue I had that confused me the most. In JEI it tells me the items I need for making different cheese, but when I put salt and rotten flesh, Asiago, when I used salt and grey dye, Asiago... tried a few different recipes and the results are always the same for some reason.

>  Which recipes? The GUI and JEI/recipes have nothing to do with each other, so need to know which items started showing up in JEI.

Will try tomorrow, I just remember having only two roaster recipes before configurating the file.

> This is expected as putting too much yeast in a fermenting barrel will spoil the extra yeast and make it not usable.

While this makes sense in real life, this differs from what ppl are used to in the game, because MC GUI´s normally takes items one by one, like the furnace does. If this is going to stay this way, it would be helpful for ppl to have this fact explained in the in-game book, bc some ppl could lose 64 yeast just bc they dont know much about GC or they could think they need a great amount of yeast for fermenting fluids.
